packages/babel-types/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "dependencies": {
     "@babel/helper-validator-identifier": "workspace:^",
-    "to-fast-properties": "^2.0.0"
+    "to-fast-properties": "condition:BABEL_8_BREAKING ? ^4.0.0 : ^2.0.0"
```

Comment:
v4 is native ESM, that we can't use yet because we compile to CJS. We should downgrade to v3.

Reply:
🤔 Aren't we gonna make Babel 8 ESM-only?

Reply:
Yes, but it's not ready yet so the current Babel 8 build fails with this PR :stuck_out_tongue: 

We will be able to upgrade before Babel 8.0.0, but not yet.